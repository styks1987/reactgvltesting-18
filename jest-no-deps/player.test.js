/** @format */

const player = require('./player');

test('default player specs', () => {
    expect(player.getHuman()).toMatchSnapshot();
});

test('test setting super human', () => {
    player.setSuperHuman(true);
    expect(player.getHuman()).toMatchSnapshot();
});

test('test setting is lost', () => {
    player.setLost(true);
    expect(player.getHuman()).toMatchSnapshot();
});

test('test setting name', () => {
    player.setName('Tom');
    expect(player.getHuman()).toMatchSnapshot();
    expect(player.getHuman().name).toBe('Tom');
});
