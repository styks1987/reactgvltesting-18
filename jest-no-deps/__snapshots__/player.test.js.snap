// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default player specs 1`] = `
Object {
  "isLost": undefined,
  "name": undefined,
  "superhuman": undefined,
}
`;

exports[`test setting is lost 1`] = `
Object {
  "isLost": true,
  "name": undefined,
  "superhuman": true,
}
`;

exports[`test setting name 1`] = `
Object {
  "isLost": true,
  "name": "Tom",
  "superhuman": true,
}
`;

exports[`test setting super human 1`] = `
Object {
  "isLost": undefined,
  "name": undefined,
  "superhuman": true,
}
`;
